### PR TITLE
 Add: version checking for Blur effect, path information for error message.

### PR DESCRIPTION
**Pull request by simeoncran**
Blur effects will only work on UAP version 8 and above. Previously if you tried to run LottieGen to generate code that could run on version 7 and the Lottie file used blur, we'd throw an exception. Now we report the issue and turn off the blur.

Designers have asked for more context for the LT0035 error message, so I've added information about the layer and ancestor precomp layers. This information could be added to other issue messages in future.

### CODE REVIEW

File: source/LottieToWinComp/PreComps.cs [hunk 1]
```diff
@@ -211,13 +211,21 @@ static LayerVisual ApplyDropShadow(PreCompLayerContext context, Visual visual, D
...
+            ContainerVisual source,
             GaussianBlurEffect gaussianBlurEffect)
         {
             Debug.Assert(gaussianBlurEffect.IsEnabled, "Precondition");
 
+            var factory = context.ObjectFactory;
+
+            if (!factory.IsUapApiAvailable(nameof(CompositionVisualSurface), versionDependentFeatureDescription: "Gaussian blur"))
+            {
+                // The effect can't be displayed on the targeted version.
```
Comment (scott-moore-ms) [inline source/LottieToWinComp/PreComps.cs:225]: >// The effect can't be displayed on the targeted version. [](start = 16, length = 57)

Should this report some sort of warning?

**Review by scott-moore-ms — APPROVED** [review]
:shipit:

**Review by derobinsMSFT — APPROVED** [review]
:shipit:

File: source/LottieToWinComp/PreComps.cs (hunk 1 above)
Comment (simeoncran) [reply source/LottieToWinComp/PreComps.cs:225]: It does. IsUapApiAvailable complains.

---
In reply to: [537666412](https://github.com/windows-toolkit/Lottie-Windows/pull/404#discussion_r537666412) [](ancestors = 537666412)

**Pull request merged**